lib/idp_common_pkg/idp_common/classification/service.py

```diff
@@ -2147,4 +2147,4 @@ def holistic_classify_document(self, document: Document) -> Document:
             )
             raise
 
-        return document
```

Comment:
Try to get rid of all the 'noisy' diffs..  makes it much harder to review the important changes.